nulib/dc-api-v2
# Remove add annotations workaround in IIIF response

## What does this do?
In a IIIF API response:

- Updates `iiif-builder` package to latest version, which includes support for applying `annotations` property onto a `canvas`.
- Removes our manual workaround of applying  `annotations` to a `canvas`

## How to test
- In the dev environment, identify (or create) a Work of "Video" type in your Meadow app which contains multiple Filesets.  
- In Meadow, under a Work's Structure tab, click "Edit Structure (VTT)" button and add some valid VTT code.
- Inspect this Work directly in the DC API at `http://localhost:3000/works/WORK_ID_HERE?as=iiif, and verify that the Fileset (`canvas`) with the above VTT has `annotations` filled out.
